library/global.ml

```diff
@@ -9,7 +9,6 @@
 (************************************************************************)
 
 open Names
-open Environ
```

Comment:
not sure about thiss change

Reply:
There were some explicit module prefixes in the file already with this. I think it makes sense since many names are shadowed in this file anyway.